hfc.py

```diff
@@ -22,34 +31,47 @@ def __str__(self):
     
 """)
 
+#variable to hold the github username
     gh_name = input("Please enter your github username: ")
 
+#variable req to hold data about the issues created/opened
```

Comment:
Same as the comment on line 15. Please fix and change to:
`Request object holding PR data from the github API.`